pkg/apis/pipeline/v1alpha1/run_types_test.go

```diff
@@ -172,6 +172,27 @@ func TestRunIsCancelled(t *testing.T) {
 	if !run.IsCancelled() {
 		t.Fatal("Expected run status to be cancelled")
 	}
+	expected := ""
+	if string(run.GetStatusMessage()) != expected {
+		t.Fatalf("Expected StatusMessage is %s but got %s", expected, run.GetStatusMessage())
+	}
+}
+
+func TestRunIsCancelledWithMessage(t *testing.T) {
```

Comment:
It may be worth validating that the statusMessage is not set unless the status is set (in run_validation)